hid/pico/src/ph_ps2.c

```diff
@@ -123,13 +126,13 @@ void ph_ps2_mouse_send_wheel(s8 h, s8 v) {
 
 void ph_ps2_send_clear(void) {
 	if (PH_O_IS_KBD_PS2) {
-		for(u8 key = 0xe0; key <= 0xe7; key++) {
-			kb_send_key(key, false, 0);
-		}
+		//for(u8 key = 0xe0; key <= 0xe7; key++) {
```

Comment:
BTW why you commented this?

Reply:
This doesn't work. The queue is only 8 bytes long, so blasting it with circa 200 bytes of key-up events won't do it. I'll have to think about key tracking so only those keys which where held down get reset with key-up events.

Reply:
Got it.